packages/new-editor/src/components/editor/Editor.css

```diff
@@ -1,3 +1,7 @@
+.editor {
+  height: 100vh !important;
```

Comment:
sorry but I don't like to see `!important` statements in CSS...
IMO this is mostly not needed and only a sign for missing knowledge how CSS and the specificity of selectors work: https://1ivy.atlassian.net/wiki/spaces/IVYTEAM/pages/6988639/CSS